calibration/CTDMOG/CGINS-CTDMOG-13598__20170402.csv

```diff
@@ -0,0 +1,25 @@
+serial,name,value,notes
+37-13598,CC_a0,-1.511576e-04,source_file 3305-00101-00529-A_SN_37-13598_QCT_Results_CTDMO-G.log
+37-13598,CC_a1,3.174736e-04,
+37-13598,CC_a2,-5.286432e-06,
+37-13598,CC_a3,2.239171e-07,
+37-13598,CC_cpcor,-9.57E-08,
+37-13598,CC_ctcor,3.25E-06,
+37-13598,CC_g,-9.892839e-01,
+37-13598,CC_h,1.371966e-01,
+37-13598,CC_i,-1.577688e-04,
+37-13598,CC_j,3.134907e-05,
+37-13598,CC_p_range,1450,
+37-13598,CC_pa0,1.092235e-02,
+37-13598,CC_pa1,4.508792e-03,
+37-13598,CC_pa2,-2.848615e-11,
+37-13598,CC_ptca0,5.243820e+05,
+37-13598,CC_ptca1,-2.504053e+00,
+37-13598,CC_ptca2,1.241916e-01,
+37-13598,CC_ptcb0,2.508212e+001,
+37-13598,CC_ptcb1,2.250000e-004,
+37-13598,CC_ptcb2,0.00E+00,
+37-13598,CC_ptempa0,1.832560e+02,
+37-13598,CC_ptempa1,-6.713872e-02,
+37-13598,CC_ptempa2,-5.566722e-07,
+37-13598,CC_wbotc,1.055593e-06,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT